src/sentry/integrations/utils/code_mapping.py

```diff
@@ -116,10 +116,13 @@ def _find_code_mapping(self, frame_filename: FrameFilename) -> Union[CodeMapping
                         self.trees[repo_full_name], frame_filename
                     )
                 )
-            except Exception:
-                logger.exception(
-                    f"Code mapping failed for {frame_filename} in {repo_full_name}. Processing continues."
+            except NotImplementedError:
```

Comment:
We should have not been getting NotImplementedError anymore, however, we were because `path/foo.py` was being considered and raising this.

The errors should dissappear after this PR.